### ISSUE TITLE
Uncaught TypeError: Error processing argument at index 0, conversion failure from null

### ISSUE THREAD

**Issue by sbilaldogan** [atom/deprecation-cop#68]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.8.0
**System**: Microsoft Windows 10 Pro
**Thrown From**: [deprecation-cop](https://github.com/atom/deprecation-cop) package, v0.54.1
### Stack Trace

Uncaught TypeError: Error processing argument at index 0, conversion failure from null

```
At C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:189

TypeError: Error processing argument at index 0, conversion failure from null
    at TypeError (native)
    at openExternally (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:189:33)
    at Object.$.ajax.success (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:203:22)
    at fire (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3099:30)
    at Object.self.fireWith [as resolveWith] (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3211:7)
    at done (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8264:14)
    at XMLHttpRequest.<anonymous> (C:\Users\Nuray\AppData\Local\atom\app-1.8.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8605:9)
```
### Commands

```
     -1:29.1.0 linter:lint (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
angular-2-typeScript-snippets, v0.4.6
atom-typescript, v8.11.0
linter, v1.11.14

# Dev
No dev packages
```

**Comment by Stanzilla** [comment]
Happened to me today

**Comment by Ben3eeE** [comment]
Get directed here when clicking `open issue` in the Deprecation cop on `atom-material-syntax` a few times, it also errors in the console.
![depcop](https://cloud.githubusercontent.com/assets/1058982/20190378/77e0b13a-a781-11e6-8af9-38da4d404658.gif)

**Comment by Stanzilla** [comment]
this is still happening with 1.14beta0

**Comment by Spidersouris** [comment]
yup, just happened to me

**Comment by karneaud** [comment]
+1

**Comment by NishL** [comment]
Happening on v1.13.0, Windows 10.

**Comment by TomerHeber** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.13.0
**Electron Version**: 1.3.13
**System**: Microsoft Windows 10 Pro
**Thrown From**: [deprecation-cop](https://github.com/atom/deprecation-cop) package, v0.55.1


### Stack Trace

Uncaught TypeError: Error processing argument at index 0, conversion failure from null

```
At C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:164

TypeError: Error processing argument at index 0, conversion failure from null
    at TypeError (native)
    at openExternally (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:164:33)
    at Object.success (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:178:22)
    at fire (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3099:30)
    at Object.fireWith [as resolveWith] (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3211:7)
    at done (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8264:14)
    at XMLHttpRequest.<anonymous> (C:\Users\Tomer\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8605:9)
```

### Commands

```
  2x -0:20.7.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "telemetryConsent": "limited"
  }
}
```

### Installed Packages

```coffee
# User
angularjs, v0.3.5 (active)
autocomplete-go, v1.3.0 (active)
builder-go, v1.0.1 (active)
go-config, v1.2.4 (active)
go-debug, v1.0.3 (active)
go-get, v2.1.1 (active)
go-plus, v5.1.0 (active)
godoc, v1.1.1 (active)
gofmt, v1.2.0 (active)
gometalinter-linter, v1.2.2 (active)
gorename, v1.1.0 (active)
linter, v1.11.21 (active)
navigator-go, v1.1.3 (active)
tester-go, v2.0.2 (active)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (active)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (active)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.85.1 (active)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.1 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (active)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by cxt666** [comment]
**Atom Version**: 1.13.0
**Electron Version**: 1.3.13
**System**: P02034635
**Thrown From**: [deprecation-cop](https://github.com/atom/deprecation-cop) package, v0.55.1


### Stack Trace

Uncaught TypeError: Error processing argument at index 0, conversion failure from null

```
At C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:164

TypeError: Error processing argument at index 0, conversion failure from null
    at TypeError (native)
    at openExternally (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:164:33)
    at Object.success (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\deprecation-cop\lib\deprecation-cop-view.js:178:22)
    at fire (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3099:30)
    at Object.fireWith [as resolveWith] (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:3211:7)
    at done (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8264:14)
    at XMLHttpRequest.<anonymous> (C:\Users\menzemth\AppData\Local\atom\app-1.13.0\resources\app.asar\node_modules\jquery\dist\jquery.js:8605:9)
```

### Commands

```
     -1:20.2.0 core:copy (span.syntax--punctuation.syntax--md)
     -1:14.5.0 core:paste (div.line)
     -1:10.4.0 core:copy (span.syntax--text.syntax--md)
     -1:06.1.0 core:paste (div.line)
     -0:50.4.0 command-palette:toggle (input.hidden-input)
  2x -0:47.8.0 core:move-down (input.hidden-input)
     -0:47.2.0 core:confirm (input.hidden-input)
     -0:47.2.0 sync-settings:restore (input.hidden-input)
  2x -0:33.3.0 deprecation-cop:view (atom-workspace.workspace.scrollbars-visible-always.compact-tab-bar.use-animations.theme-atom-material-syntax.theme-atom-material-ui)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "language-gfm"
    ],
    "telemetryConsent": "no",
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-material-syntax, v0.4.8 (active)
atom-material-ui, v1.3.9 (active)
build, v0.67.0 (active)
build-powershell, v0.8.2 (active)
busy, v0.7.0 (active)
language-markdown, v0.18.2 (active)
language-powershell, v4.0.0 (active)
pp, v0.0.1 (active)
pp-markdown, v0.0.2 (active)
script, v3.14.0 (inactive)
sync-settings, v0.8.1 (active)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (inactive)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.14.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.33.1 (active)
autocomplete-snippets, v1.11.0 (active)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (active)
background-tips, v0.26.1 (active)
bookmarks, v0.43.2 (active)
bracket-matcher, v0.85.1 (active)
command-palette, v0.39.1 (active)
deprecation-cop, v0.55.1 (active)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (active)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.2.0 (active)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.1 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.159.1 (active)
metrics, v1.1.2 (active)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (active)
snippets, v1.0.4 (active)
spell-check, v0.68.5 (active)
status-bar, v1.6.0 (active)
styleguide, v0.48.0 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (active)
timecop, v0.33.2 (active)
tree-view, v0.211.1 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (active)
whitespace, v0.35.0 (active)
wrap-guide, v0.39.0 (active)
language-c, v0.54.0 (active)
language-clojure, v0.22.1 (active)
language-coffee-script, v0.48.1 (active)
language-csharp, v0.13.0 (active)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (active)
language-go, v0.43.0 (active)
language-html, v0.47.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (active)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (active)
language-mustache, v0.13.0 (active)
language-objective-c, v0.15.1 (active)
language-perl, v0.37.0 (active)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (active)
language-ruby, v0.70.2 (active)
language-ruby-on-rails, v0.25.1 (active)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (active)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by as-cii** [comment]
Thanks for the reports and apologies for the regression. I am unable to reproduce on macOS with the reproduction steps provided by @Ben3eeE so I suspect this might be related to some nuance in how deprecation reporting is handled on Windows. 

@damieng: thoughts? 💭

**Comment by madakk** [comment]
Looks like the fault is with shortenURL, which only takes place on Windows platforms. The shorten URL call in this case returns a 400 response code with the text "Error: Please enter a valid URL to shorten". The shortenURL call then returns null, which I guess is the cause of this error.

**Comment by damieng** [comment]
Additionally shortenURL is using git.io - we typically use is.gd these days such as in the notification package.  I'll take a look.

**Comment by batjko** [comment]
I'm not sure this should have been closed.
#78 Is not addressing the OP's issue and the package still uses deprecated styles, according to Atom 1.13.0.

**Comment by madakk** [comment]
> I'm not sure this should have been closed.
> #78 Is not addressing the OP's issue and the package still uses deprecated styles, according to Atom 1.13.0.

This issue is about an error thrown when trying to open an issue from the deprecation cop view, it's not about any package using deprecated styles.

**Comment by batjko** [comment]
@madakk I noticed my mistake and removed my comment immediately. Apparently not quickly enough, though.

**Comment by lewismoten** [comment]
I'm still getting this error reported in Atom 1.13.1
```

The contents of atom-text-editor elements are no longer encapsulated within a shadow DOM boundary. Please, stop using shadowRoot and access the editor contents directly instead.
Report Issue
atom-text-editor.Object.defineProperty.get - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\src\text-editor-element.js:68:18
<unknown> - C:\Users\lewismoten\.atom\packages\css-color-underline\lib\main.coffee:14:29
Function.module.exports.Emitter.simpleDispatch - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14
Emitter.emit - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:129:28
<unknown> - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\src\text-editor.js:541:32
Function.module.exports.Emitter.simpleDispatch - C:\Users\lewismoten\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\emitter.js:25:14
```

**Comment by damieng** [comment]
Yes, the fix is part of 1.14.
